docs/PARAMETERS.md

```diff
@@ -36,11 +36,11 @@ Transmission power in Watts.
 
 #### Tx Diversity #### 
 Diversity mode. 
-Can be "enabled", "antenna1", "antenna2". 
+Can be "enabled", "antenna1", "antenna2", "r:en, t:ant1", "r:en, t:ant2".
```

Comment:
I guess could be useful if there would be some sort of brief explanation. If not here, maybe somewhere else?
No blocker for sure.

Reply:
Good idea - added a line for both Tx and Rx parameters explaining what that does.